roles/upload-site/tasks/main.yml

```diff
@@ -108,6 +108,8 @@
     - 'ev "_paranoia_remove_risky_permissions();"'
     - "-y --exact vset sws_migration_version {{ migration_version }}"
     - "-y --exact vset stanford_ssp_role_map_source workgroup"
+    - "-y --exact vset stanford_ssp_workgroup_api_cert /mnt/www/html/cardinald7.02live/simplesamlphp/cert/stanford_ssp.cert"
+    - "-y --exact vset stanford_ssp_workgroup_api_key /mnt/www/html/cardinald7.02live/simplesamlphp/cert/stanford_ssp.key"
```

Comment:
Needs to have environment detection. Can't set these to 02live on dev or test.

Reply:
See https://github.com/SU-SWS/ansible-playbooks/blob/master/roles/define-paths/tasks/main.yml#L44